content/posts/02_staff_expectations/index.md

```diff
@@ -0,0 +1,116 @@
+their own work done and make sure others can do it too. They should help other engineers and teams manage and prioritize complex projects. 
+They should act as a "glue" to ensure engineering teams are working and delivering smoothly. They should observe and eliminates the developer
+experience problems and increase productivity.
+
+Some of the examples of getting stuff done:
+* Help the team with project management and prioritization.
+* Identify hidden risks and dependencies, which could disturb project delivery.
+* Advocate with leadership and find the necessary resource to finish a project.
+* Find a way to solve a particularly difficult problem.
+
+## Be a role model and mentor others
+![](./arrows.png)
+
+At some point in a career, you usually start to have a bigger impact through others, rather than by your individual contributions. This 
+happens not only on the management track but also for technical leaders. Some foretaste comes even for senior engineers, but you get a full
+experience of it in staff+ positions. 
+
+Younger people start to look up to staff engineers and even try to mimic their behavior. Staff engineers become role models for
+others. That's why it's important to demonstrate the company's values and culture through actions. They get the leverage to align people and 
+multiply the outcome.
+
+Staff engineers also get an opportunity (and obligation) to 
+[mentor](https://blog.pragmaticengineer.com/developers-mentoring-other-developers/) other people. We consider it a great privilege, to be 
+able to pass your experiences and your knowledge to other people. Helping people grow and succeed is very important. Not only you're 
+actively using your time for others, but also you're increasing the area of impact. We also believe, that during mentoring, you can learn 
+a tremendous amount of things about yourself, efficient communication, setting goals, etc.
+
+Examples, of how to influence others:
+* Participate in or start a mentoring program at your company.
+* Actively demonstrate company values and culture through your actions
+* Review code, documents, proposals, etc.
+* Give feedback!
+
+## Communicate clearly and bring clarity
+![](./communication.png)
+
+Miscommunication is often a source of most problems at work (and in life). Misunderstanding can result in poor performance, subpar
+quality, stressed people, or wasted time. The role of staff engineers is to bring 
+[clarity and alignment](https://alexewerlof.substack.com/p/6-ways-staffs-reach-clarity).
+
+There are several things staff engineers should do to improve communication in their teams and company:
+* Listen carefully to others and understand their problems and point of view.
+* Make sure the team reaches a collective understanding.
+* [Start with why](https://simonsinek.com/books/start-with-why/) and explain why we work on certain things.
+* Lead by example and improve their communication skills. Show more junior engineers this is often a key to success.
```

Comment:
I'd add one super important point for me.
`Identify and challenge the X/Y problem (https://xyproblem.info/).`

If I were to describe the trait of my work it would be a strong taste on identifying them. It's notorious that people don't start with why and hance try to solve the outcome, not the rootcause.